lib/gollum-lib/markups.rb

```diff
@@ -59,5 +65,8 @@ class Markup
              :extensions => ['mediawiki','wiki'], :reverse_links => true)
     register(:pod, "Pod",
              :enabled => MarkupRegisterUtils::executable_exists?("perl"))
+    register(:bib, "BibTeX", :extensions => ['bib'],
+             :enabled => MarkupRegisterUtils::gems_exist?(["bibtex-ruby", "citeproc-ruby", "csl-styles"]),
```

Comment:
As a general remark: can end users easily figure out why `:enabled` would be false? For example, say that I have failed to install `citeproc-ruby`, how would I know that this is the case?